libs/expo/betterangels/src/lib/screens/ClientHMIS/tabs/ClientInteractionsHmisView/ClientInteractionsHmisView.tsx

```diff
@@ -30,7 +34,10 @@ export function ClientInteractionsHmisView(props: TProps) {
       document: HmisNotesDocument,
       queryFieldName: 'hmisNotes',
       pageSize: DEFAULT_PAGINATION_LIMIT,
-      variables: { filters: { hmisClientProfile: client?.id } },
+      variables: {
+        filters: { hmisClientProfile: client?.id },
+        ordering: { date: Ordering.Desc },
```

Comment:
we should include a secondary ordering param. e.g., 
```suggestion
        ordering: [{ date: Ordering.Desc }, { id: Ordering.Desc }]
```

especially important here, to keep results consistent, since we only have date, not datetime